### PR TITLE
docs(language-service): Clarify WebStorm installation instructions

### PR DESCRIPTION
**Pull request by fkorak**
The instructions for WebStorm don't match the official docs anymore. JetBrains seems to have a much easier setup for their IDE now. Updated the documentation to reflect that.

This is a second attempt at a pull request, after I broke something trying to rebase, and did not know how to recover.

## PR Checklist
Please check if your PR fulfills the following requirements:

- [X] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [X] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

### CODE REVIEW

File: aio/content/guide/language-service.md [hunk 1]
```diff
@@ -64,36 +64,9 @@ In the marketplace, search for Angular Language Service extension, and click the
...
-
-*OR*
-
-```sh
-yarn install
-```
-
-When Angular sees this dev dependency, it provides the language service in the WebStorm environment.
-WebStorm then gives you colorization inside the template and autocomplete in addition to the Angular Language Service.
+[WebStorm](https://www.jetbrains.com/webstorm/) automatically bundles an [extension](https://plugins.jetbrains.com/plugin/6971-angular-and-angularjs) which has the same feature set as the Angular Language Service. In other JetBrain products the plugin can be installed through the marketplace. Make sure that you have AngularJS or Angular library files in your project (for example, in the node_modules folder) – this is required for the IDE to enable Angular support for this project.
```
Comment (kyliau) [inline aio/content/guide/language-service.md:67]: > In other JetBrain products

Do you mean "Like other JetBrain products"?

**Comment by kyliau** [comment]
@fkorak thanks for reopening the PR. The commit message should be updated to conform to our commit format.
Could you please change it to:
```
docs(language-service): Clarify WebStorm installation instructions
```

File: aio/content/guide/language-service.md (hunk 1 above)
Comment (fkorak) [reply aio/content/guide/language-service.md:67]: The [extension](https://plugins.jetbrains.com/plugin/6971-angular-and-angularjs) is listed as compatible with IntelliJ IDEA, AppCode, CLion, GoLand, PhpStorm, PyCharm, Rider, RubyMine and WebStorm, which would mean almost all of the Jetbrains IDE products (DataGrip, their SQL IDE, is not listed). I checked IntelliJ Ultimate and it gets bundled per default, but I don't have access to check all of their other products. But it seems to me the plugin can be installed in most other JetBrain products. 
Should all compatible IDEs be listed separately in the angular docs? WebStorm is probably the main one people will care about.

**Comment by fkorak** [comment]
> @fkorak thanks for reopening the PR. The commit message should be updated to conform to our commit format.
> Could you please change it to:
> 
> ```
> docs(language-service): Clarify WebStorm installation instructions
> ```

Done.

File: aio/content/guide/language-service.md (hunk 1 above)
Comment (kyliau) [reply aio/content/guide/language-service.md:67]: Ah I see what you mean now. We don't have to list all JetBrain products, but perhaps one or two prominent products will be helpful.

> In other JetBrain products (IntelliJ IDEA, AppCode, etc) ...

**Comment by kyliau** [comment]
Please update the original commit description (not the PR description) so that the lint check could pass.

If you're unfamiliar with git, this can be done by first amending the message
```
git commit --amend --message "docs(language-service): Clarify WebStorm installation instructions"
```
then force push to your branch
```
git push --force
```

File: aio/content/guide/language-service.md (hunk 1 above)
Comment (fkorak) [reply aio/content/guide/language-service.md]: > 
> 
> Ah I see what you mean now. We don't have to list all JetBrain products, but perhaps one or two prominent products will be helpful.
> 
> > In other JetBrain products (IntelliJ IDEA, AppCode, etc) ...

I added it.

**Comment by fkorak** [comment]
> 
> 
> Please update the original commit description (not the PR description) so that the lint check could pass.
> 
> If you're unfamiliar with git, this can be done by first amending the message
> 
> ```
> git commit --amend --message "docs(language-service): Clarify WebStorm installation instructions"
> ```
> 
> then force push to your branch
> 
> ```
> git push --force
> ```

I tried doing that, and it ran the tests again, so I'm hoping it worked?

**Comment by AndrewKushnir** [comment]
@fkorak it looks like there are 2 commits in your PR, could you please squash them into one (keeping the `docs(language-service): Clarify WebStorm installation instructions` one)? Also there is conflict with the main branch, could you please rebase and resolve the conflict? Thank you.

**Comment by AndrewKushnir** [comment]
Closing this PR due to the lack of reply. Please reopen if needed. Thank you.

**Pull request closed without merging**